tests/Backend/SOX/MergeRequestsTest.php

```diff
@@ -797,6 +797,139 @@ public function testConfigurationUpdatedInBranch(): void
         $this->assertBranchIsDeleted($newBranch['id']);
     }
 
+    public function testVersionIdentifiersWhenCreatedConfigurationInBranch(): void
```

Comment:
dalo by se to spojit do jednoho testu, nebo k necemu prilepit ale preferuju spis, aby to bylo explicitne videt, oba usecasy. Jinak to zvysuje riziko, ze mozek samovolne opusti prostor lebky, familierne nazyvane jako "bouchne mi z toho hlava"

Reply:
hmm, tak na to koukam a vlastne jsem uplne zrcadlil testy testCreateConfigurationInBranch a testConfigurationUpdatedInBranch . ASi by bylo lepsi jim pridat check na changeLog a realizovat ten test tam, misto pridavani techto dovu

Reply:
No jak chces uz by som to mozno nechal ked to mas napisane